### PR TITLE
Adds contributing guidelines.

### PR DESCRIPTION
**Pull request by LamboCreeper**
Makes it more clear to people who wish to contribute to the project.

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Contributing Guidelines
+
+If you are considering contributing please read through this document, if you have any questions do not hesitate to ask in the CodeSupport Discord.
```
Comment (jacobgasser) [inline .github/contributing.md]: "If you are considering contributing, please read through this document. If you have any questions please do not hesitate to ask in the <link somehow>CodeSupport Discord</link somehow>"

File: .github/contributing.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Contributions to the Discord Bot **must** pass our ESLint check and be approved by at least one person before being merged.
+
+## Contributing Steps
+1. Fork the repository
+2. Create a new branch using the format `feature/<short-summary>`
+    - For example, if you were adding a hello world command: `feature/add-hello-world-command`
+3. Pull in develop, do not base your changes off of any other branch
+4. Make any changes you wish
+5. Create a pull request, merging your branch back into the CodeSupport Discord Bot repository's develop branch.
```
Comment (jacobgasser) [inline .github/contributing.md]: Remove period

File: .github/contributing.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Pull Requests
+- Make small feature based pull requests, this makes it easier to read
+- The title of your pull request should give a good summary of the change
+- The description of your pull request should detail what has changed, along with any information people should be aware of
+
```
Comment (jacobgasser) [inline .github/contributing.md:20]: Add something about including your discord name

**Review by jacobgasser — CHANGES_REQUESTED** [review]

File: .github/contributing.md (hunk 3 above)
Comment (LamboCreeper) [reply .github/contributing.md:20]: Good point! This would be super handy.

**Review by jacobgasser — APPROVED** [review]

**Review by jacobgasser — APPROVED** [review]

**Pull request merged**